Error: `aria-labelledby` must reference one (or more) valid element `id` attributes.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
diff --git a/output/Checkbox Group/4/0f49d4e9-06ae-4843-bece-f614a639262f.html b/output/Checkbox Group/4/0f49d4e9-06ae-4843-bece-f614a639262f.html
index a535d7a..9fff873 100644
--- a/output/Checkbox Group/4/0f49d4e9-06ae-4843-bece-f614a639262f.html	
+++ b/output/Checkbox Group/4/0f49d4e9-06ae-4843-bece-f614a639262f.html	
@@ -1,6 +1,15 @@
-<label>What fruits do you like?</label>
+<label id="fruits-label">What fruits do you like?</label>
 <div role="group" aria-labelledby="fruits-label">
-  <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-  <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-  <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
 </div>
\ No newline at end of file
